src/main/java/meteordevelopment/meteorclient/systems/modules/world/AirPlace.java

```diff
@@ -99,4 +105,29 @@ private void onRender(Render3DEvent event) {
 
         event.renderer.box(((BlockHitResult) hitResult).getBlockPos(), sideColor.get(), lineColor.get(), shapeMode.get(), 0);
     }
+
+    private void setHitResult() {
+        final double r = customRange.get() ? range.get() : mc.interactionManager.getReachDistance();
+        for (int i = (int) r; i > 0; i -= 1D) {
+            hitResult = mc.getCameraEntity().raycast(Math.min(r, i), 0, false);
```

Comment:
Why do you need this part though? Doesn't this just ray cast but with decrementing lengths?

Also why do you need to use `Math.min(r, i)` when `i` will always be equal or less than `r`, just use `i` instead.